marcdown_project/marcdown/models/note.py

```diff
@@ -21,6 +21,9 @@ class Note(models.Model):
     def __str__(self):
         return self.title
     
+    def is_owner(self, user):
+        return hasattr(user, 'profile') and self.owner == user.profile
```

Comment:
Perhaps `user.is_authenticated` would be more relevant and readable than the `hasattr`